source/Marquee.js

```diff
@@ -232,6 +243,18 @@
 		* @method
 		* @private
 		*/
+		destroy: enyo.inherit(function (sup) {
+			return function () {
+				if (_getMarqueeOnHoverControl()) 
```

Comment:
This absolutely must be a test -> `if (this === _getMarqueeOnHoverControl())` we can't just set it to null because it was destroyed and some control existed the `this` may not be the one that was set!